docs/getting-started/concepts/permission.md

```diff
@@ -0,0 +1,111 @@
+# Permission
+
+## Overview
+
+The Feast permissions model allows to configure granular permission policies to all the resources defined in a feature store.
+
+The configured permissions are stored in the Feast registry and accessible through the CLI and the registry APIs.
+
+The permission authorization enforcement is performed when requests are executed through one of the Feast (Python) servers
+- The online feature server (REST)
+- The offline feature server (Arrow Flight)
+- The registry server (gRPC)
+
+Note that there is no permission enforcement when accessing the Feast API with a local provider.
+
+## Concepts
+
+The permission model is based on the following components:
+- A `resource` is a Feast object that we want to secure against unauthorized access.
+  - We assume that the resource has a `name` attribute and optional dictionary of associated key-value `tags`.
+- An `action` is a logical operation executed on the secured resource, like:
+  - `create`: Create an instance.
+  - `describe`: Access the instance state.
+  - `update`: Update the instance state.
+  - `delete`: Delete an instance.
+  - `query`:  Query both online and offline stores.
```

Comment:
shouldn't this be `read`?

Reply:
looks like we misunderstood each other. because `read` can be interpreted as both "access state" and "read data", my suggestion was not to use `read` anymore at all. Instead we will have:

- `describe` - this means that a user is allowed to read info from the registry about the feature view.
- `query` - this means that a user is allowed to read dataset from online/offline stores.

cc @dmartinol

Reply:
got it. I'm okay with that, it just doesn't fit the CRUD convention, right?

Reply:
yeah, it doesn't, but results in less ambiguous naming, which was the goal. plus, describe is also widely used in such cases (in aws iam, at least), so I think that should be fine as well.